benchmark/src/main/scala/fs2/benchmark/StreamBenchmark.scala

```diff
@@ -48,10 +51,9 @@ class StreamBenchmark {
       .unsafeRunSync
       .get
 
-  @GenerateN(1, 10, 100, 1000, 10000, 100000)
   @Benchmark
-  def rightAssocFlatMap(N: Int): Int =
-    (0 until N)
+  def rightAssocFlatMap(n: Int): Int =
```

Comment:
shouldn't this benchmark use the member Param `n` as well?

Reply:
Yep, fixed